packages/aws-sdk-signers/src/aws_sdk_signers/signers.py

```diff
@@ -348,7 +353,7 @@ def _should_sha256_sign_payload(
         if request.destination.scheme != "https":
             return True
 
-        return signing_properties.get("payload_signing_enabled", True)
+        return signing_properties.get("payload_signing_enabled", False)
```

Comment:
I think this still needs to default to `True` for now.  `UNSIGNED-PAYLOAD` is only supported for operations with the [@unsignedPayload trait](https://smithy.io/2.0/aws/aws-auth.html#aws-auth-unsignedpayload-trait).

For now, none of the services we need to support have the unsignedPayload trait on any operations, so I think the best course is to revert this default to `True`.

Long term - the auth resolver will likely need to check the runtime model for the operation to inspect the traits and set this property correctly per operation. (Note - I've created an entry in our tracker for this).